contributed_definitions/NXmpes_arpes.nxdl.xml

```diff
@@ -334,21 +334,28 @@ with higher granularity in the data description.
                 </field>
             </group>
         </group>
-        <group name="data" type="NXdata_mpes">
+        <group name="data" type="NXdata">
             <attribute name="signal">
+                <doc>
+                     There is an object named data that contains the signal.
+                </doc>
                 <enumeration>
                     <item value="data"/>
                 </enumeration>
             </attribute>
             <attribute name="axes">
+                <doc>
+                     There are three dimensions, one energy and two angular coordinates. Any coordinates that do not move,
+                     are represented by one point.
+                </doc>
                 <enumeration>
-                    <item value="[angular0, angular1, energy]"/>
+                    <item value="['angular0', 'angular1', 'energy']"/>
```

Comment:
This is probably more a question to @rettigl. But is this really working? This would just check if there is a string which states `"['angular0', 'angular1', 'energy']"` but the `@axes` documentation states explicitly that it needs to be an array of strings:

<img width="806" alt="Bildschirmfoto 2024-06-04 um 11 35 02" src="https://github.com/FAIRmat-NFDI/nexus_definitions/assets/1014422/09ab3de0-af8b-4bd9-ad2f-acbaf9c84694">

This would require users to follow a wrong nexus notation. But probably there is no alternative to doing something like this in nexus.